src/blacklist.txt

```diff
@@ -50,4 +50,7 @@ wss://*.2giga.link/wproxy*
 *://*.rocks.io/proxy*
 *://ad-miner.com/lib/*
 *://*.ad-miner.com/lib/*
-*://party-nngvitbizn.now.sh/*
\ No newline at end of file
+*://party-nngvitbizn.now.sh/*
+*://*.host.d-ns.ga/*
```

Comment:
This is sufficient as it covers anything including ws(s).

Reply:
Hi Rafael, thank you for your feedback. 

Unfortunately using a wildcard for the protocol will only block `http` and `https`, as documented per [Google Chrome's Match Patterns](https://developer.chrome.com/extensions/match_patterns). There has been some discussion  about this in [Bug Report 129353](https://bugs.chromium.org/p/chromium/issues/detail?id=129353). As mentioned [there (from comments 102 and on)](https://bugs.chromium.org/p/chromium/issues/detail?id=129353#c102), we need to explicitly mention `ws` and `wss` in order to block WebSocket requests.

Reply:
Sorry about that, you're right. For some reason I had in mind that the wildcard would cover all protocols but I saw that I previously added rules (for Coin Hive) for the websocket protocol. I'll resolve and merge this PR. Thanks for the submission and clarification.